api-lib/event_triggers/interactionEventToMixpanel.ts

```diff
@@ -0,0 +1,89 @@
+import * as crypto from 'crypto';
+
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+import mixpanel, { Mixpanel } from 'mixpanel';
+
+import { MIXPANEL_PROJECT_TOKEN } from '../config';
+import { errorResponse } from '../HttpError';
+import { EventTriggerPayload } from '../types';
+
+export default async function handler(req: VercelRequest, res: VercelResponse) {
+  try {
+    if (MIXPANEL_PROJECT_TOKEN == '') {
```

Comment:
maybe this should be an error that bubbles up to Sentry instead of failing silently?

Reply:
didnt want to break this on dev by throwing errors but i could see how that should work (since sentry isnt hooked up on dev)

Reply:
ok i changed this in both of the event triggers